src/components/Assets/SpecsComponent.jsx

```diff
@@ -0,0 +1,208 @@
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import { Form, Grid } from 'semantic-ui-react';
+import range from 'lodash/range';
+import DropdownComponent from '../common/DropdownComponent';
+import ArtButton from '../common/ButtonComponent';
+
+import '../../_css/AddAssetComponent.css';
+
+class SpecsComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      specs: props.specs
+    };
+  }
+
+  generateYearRange = () => {
+    const currentYear = new Date().getFullYear();
+    const firstYearToConsider = 2011;
+
+    return range(currentYear, firstYearToConsider);
+  };
+
+  generateYearOptions = () => this.generateYearRange().map(year => ({
+    key: year,
+    text: year,
+    value: year
+  }));
+
+  onSelectYearOfManufacture = (event, data) => {
+    const specs = { ...this.state.specs };
+    specs.year = data.value;
+    this.setState({ specs });
+    this.props.pickRadioValuesFromSpecsComponent(specs);
+  };
+
+  handleRadioChanges = (event, data) => {
+    const { name, value } = data;
+    const specs = { ...this.state.specs };
+    if (name === 'processorType') {
+      specs.processorType = value;
+    } else if (name === 'processorSpeed') {
+      specs.processorSpeed = value;
+    } else if (name === 'screenSize') {
+      specs.screenSize = value;
+    } else if (name === 'storage') {
+      specs.storage = value;
+    } else if (name === 'memory') {
+      specs.memory = value;
+    }
+    this.setState({ specs });
+    this.props.pickRadioValuesFromSpecsComponent(specs);
+  };
+
+  render() {
+    const screenSizes = [13, 15, 17];
+    const storageSizes = [126, 256, 512];
+    const memorySizes = [4, 8, 16];
+    const processorTypes = ['Intel core i3', 'Intel core i5', 'Intel core i7'];
+    const processorSpeeds = [1.8, 2.3, 3.0, 3.4];
```

Comment:
We have so many constants that I think it's better if we create an `enum` file that we can use to track all of them. Also, make the constants be in `CAPITAL` letters, instead of `camelCase`. This helps the user infer that they are hardcoded values